src/dsl.ts

```diff
@@ -470,6 +473,11 @@ export class SnykApiCheckDsl implements ApiCheckDsl {
       (location) => dsl.getContext(location),
       (...items) => dsl.checks.push(...items),
       (pointer: string) => jsonPointerHelpers.get(dsl.nextJsonLike, pointer),
+      (parent) =>
+        !(
+          parent.location.kind === OpenApiKind.Body ||
+          parent.location.kind === OpenApiKind.Field
+        ),
     );
```

Comment:
skip all nested added for properties so breaking change rules don't apply when request body or response body is added